docs/blog/posts/2024/07-26 asynchronous message support.md

```diff
@@ -0,0 +1,144 @@
+---
+authors:
+    - JP-Ellis
+date:
+    created: 2024-07-26
+---
+
+# Asynchronous Message Support
+
+We are excited to announce that support for verifying asynchronous message interactions has been added in the recent [release of Pact Python version 2.2.1](https://github.com/pact-foundation/pact-python/releases/tag/v2.2.1). To explore this feature, use the [`pact.v3`][pact.v3] module. A huge shoutout goes to [Val Kolovos](https://github.com/valkolovos) who contributed this feature across two very large PRs ([#714](https://github.com/pact-foundation/pact-python/pull/714) and [#725](https://github.com/pact-foundation/pact-python/pull/725)). This represents a significant step forward in the capabilities of Pact Python and on the road to full support for the Pact specification.
+
+Asynchronous messages play a crucial role in building resilient and scalable systems. They allow services to communicate with each other without blocking, which can be particularly useful when the sender and receiver are not always available at the same time. However, verifying these interactions is challenging due to the wide variety of messaging systems and protocols.
+
+Pact simplifies this process by focusing on the content of the messages rather than their transport mechanisms. This approach allows defining expected message exchanges and verifying their adherence independently of messaging systems and protocols. The Pact specification provides a way to verify these interactions, but until now, Pact Python has not supported this feature.
```

Comment:
Worth linking out to the main site page on message pact?

https://docs.pact.io/getting_started/how_pact_works#non-http-testing-message-pact

> The Pact specification provides a way to verify these interactions, but until now, Pact Python has not supported this feature.

Technically Pact python has (at least for V3 Pacts), using https://github.com/pact-foundation/pact-message-ruby but the implementation is incomplete, and has a couple of bugs

Reply:
Link added, thanks :)